sema4ai/src/sema4ai_code/vendored_deps/oauth2_settings.py

```diff
@@ -115,6 +115,12 @@ class OAuth2ProviderSettingsResolved(OAuth2ProviderSettings):
         tokenEndpoint="/oauth2/v2.0/token",
         # server=settings.server,
     ),
+    "salesforce": OAuth2ProviderSettings(
+        server="https://login.salesforce.com",
+        tokenEndpoint="/services/oauth2/token",
+        revocationEndpoint="/services/oauth2/revoke",
+        authorizationEndpoint="https://login.salesforce.com/services/oauth2/authorize",
```

Comment:
I believe you could just use relative for `authorizationEndpoint` as the other entries (so `/services/oauth2/authorize`).